<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwidget</class>
 <widget class="QWidget" name="mainwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="pTabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <widget class="QWidget" name="ptab0">
      <attribute name="icon">
       <iconset resource="Project_Wallped.qrc">
        <normaloff>:/Image/Image/clock.png</normaloff>:/Image/Image/clock.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
     <widget class="QWidget" name="ptab1">
      <attribute name="icon">
       <iconset resource="Project_Wallped.qrc">
        <normaloff>:/Image/Image/security.png</normaloff>:/Image/Image/security.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
     <widget class="QWidget" name="ptab2">
      <attribute name="icon">
       <iconset resource="Project_Wallped.qrc">
        <normaloff>:/Image/Image/message.png</normaloff>:/Image/Image/message.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
     <widget class="QWidget" name="ptab3">
      <attribute name="icon">
       <iconset resource="Project_Wallped.qrc">
        <normaloff>:/Image/Image/renewable-energy.png</normaloff>:/Image/Image/renewable-energy.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
     <widget class="QWidget" name="ptab4">
      <attribute name="icon">
       <iconset resource="Project_Wallped.qrc">
        <normaloff>:/Image/Image/settings.png</normaloff>:/Image/Image/settings.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Project_Wallped.qrc"/>
 </resources>
 <connections/>
</ui>
